### PR TITLE
List improvement

### PR DESCRIPTION
**Pull request by stef74**
#1611

### CODE REVIEW

**Comment by stef74** [comment]
capture d ecran 2017-05-04 a 20 27 43
capture d ecran 2017-05-04 a 21 17 57

Ready for review

**Comment by asaunier** [comment]
I think it's better to use the approach of PR https://github.com/c2corg/v6_ui/pull/1621

**Comment by stef74** [comment]
@asaunier rebased ... and tested.

File: less/lists.less [hunk 1]
```diff
@@ -521,21 +545,25 @@
           }
         }
 	.activity-author {
-          width: 30%;
-          align-self: center;		
+          width: 25%;
+          align-self: center;
+	  max-width: 300px;
```
Comment (asaunier) [inline less/lists.less]: Indent :)

File: less/lists.less [hunk 2]
```diff
@@ -521,21 +545,25 @@
...
 	    margin-left: 7%;
 	    margin-right: 7%;
+            align-self: center;
           }
           .outing-author {
             order: 1;
             margin-right: 20px;
             justify-content: flex-start;
+            align-self: center;	
+
```
Comment (asaunier) [inline less/lists.less]: Unneeded blank line

File: less/lists.less [hunk 3]
```diff
@@ -521,21 +545,25 @@
...
 	.activity-author {
-          width: 30%;
-          align-self: center;		
+          width: 25%;
+          align-self: center;
+	  max-width: 300px;
           .condition-rating {
             order: 0;
 	    margin-left: 7%;
 	    margin-right: 7%;
```
Comment (asaunier) [inline less/lists.less]: Indent pb with those 2 lines

File: less/lists.less [hunk 4]
```diff
@@ -492,19 +514,21 @@
           }
           .list-item-info {
             flex-direction: row;
+	    width: 100%;
```
Comment (asaunier) [inline less/lists.less]: indentation

**Pull request merged**